src/types/timestamp.rs

```diff
@@ -19,10 +23,8 @@ pub enum Precision {
     Day,
     /// Minute-level precision (e.g. `2020-08-01T12:34Z`)
     HourAndMinute,
-    /// Second-level precision. (e.g. `2020-08-01T12:34:56Z`)
+    /// Second-level precision or greater. (e.g. `2020-08-01T12:34:56Z` or `2020-08-01T12:34:56.123456789Z`)
     Second,
-    /// Sub-second precision (e.g. `2020-08-01T12:34:56.123456789Z`)
-    FractionalSeconds,
```

Comment:
🗺️ This file contains most of the changes related to removing the distinct `FractionalSeconds` variant of `Precision`.